src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -326,25 +343,227 @@ public async Task<GetTransactionModel> GetTransactionAsync(string txid)
             return model;
         }
 
+        [ActionName("listaddressgroupings")]
+        [ActionDescription("Returns a list of grouped addresses which have had their common ownership made public by common use as inputs or as the resulting change in past transactions.")]
+        public AddressGroupingModel[] ListAddressGroupings()
```

Comment:
Are not suppose to pass a wallet name? (and use default name if non provided)

Reply:
Core does not take in a wallet name, so we need to keep the method signature the same?

Reply:
ah yeah this is RPC so correct